Week3/hw/app10.js

```diff
@@ -0,0 +1,19 @@
+// 10. Use the list of vehicles to write an advertisement. So that it prints something like: `"Amazing Joe's Garage, we service cars, motorbikes, caravans and bikes."`. (Hint: use a `for` loop. The output should be correct English with all the punctuation in place (that's the challenge). So plurals for the vehicle types, commas followed by a single space, the word and to replace the final comma and closed off by a period.)
+
+let vehicle_list = ['truck', 'pickup', 'motorbike', 'caravan', 'bike'];
+let text = "Amazing Joe's Garage, we service ";
+
+function adText(param1, param2) {
```

Comment:
nice! I would only suggest changing the names of the variables.  `param1` and `param2` don't really describe much. I would suggest: `baseText` and `vehicles`

Reply:
I agree. Changed.